lib/src/utf8.dart

```diff
@@ -55,8 +55,7 @@ class Utf8 extends Struct<Utf8> {
   /// Returns a malloc-allocated pointer to the result.
   static Pointer<Utf8> toUtf8(String string) {
     final units = utf8.encode(string);
-    final Pointer<Uint8> result =
-        allocate<Uint8>(count: units.length + 1);
+    final Pointer<Uint8> result = allocate<Uint8>(count: units.length + 1);
```

Comment:
Don't we have the `dartfmt` checks enabled in `.travis.yaml`? How could this have been committed?

Reply:
We do. Could this be a change in a new version of dartfmt?

Reply:
I would say it's unlikely because such a change could cause massives amounts of user code to need re-formatting (the formatter should be very stable).

Are those checks correctly configured for travis? I see that other repositories configure it differently, e.g. https://github.com/dart-lang/stack_trace/blob/master/.travis.yml :
```
matrix:
  include:
    # Only validate formatting using the dev release
    # Formatted with 1.23.0+ which has (good) changes since 1.22.1
    - dart: dev
      dart_task: dartfmt
```

(I don't know how to correctly configure travis)